SitumWayfinding/Classes/UI/SitumMarker.swift

```diff
@@ -7,31 +7,51 @@
 
```

Comment:
Aquí máis adiante imos ter que ver como separar os elementos que se pintan no mapa do proveedor de mapas. 
O final este é un SitumMarker pero está completamente asociado a GoogleMaps
Se quixeramos usar outro proveedor teriamos ou ben que crear un OpenMapSitumMarker por exemplo ou algunha forma de relacionar os marker de situms cos dos mapas